### PR TITLE
Add basic unit tests policy

### PR DESCRIPTION
**Pull request by bmbouter**
This policy is a starting point based feedback via pulp-dev. The
language adopted was shared via pulp-dev for comment.

### CODE REVIEW

File: docs/contributing/unit_tests.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.. _tests: https://github.com/pulp/pulp/blob/3.0-dev/tests/
+
+Unit Tests
+==========
+
+All new code is highly encouraged to have basic unit tests that demonstrate its functionality. A
```
Comment (asmacdo) [inline docs/contributing/unit_tests.rst]: s/All new/New/

**Comment by bmbouter** [comment]
Thanks @asmacdo, I fixed it

**Comment by daviddavis** [comment]
Btw, it looks like you have some a commit you need to squash.

**Comment by bmbouter** [comment]
Why yes I did. Thank you.

**Pull request merged**